src/NodeApi/Runtime/NativeLibrary.cs

```diff
@@ -44,12 +44,50 @@ public static nint GetMainProgramHandle()
     public static nint Load(string libraryName)
     {
 #if NETFRAMEWORK || NETSTANDARD
-        return LoadLibrary(libraryName);
+        if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+        {
+            return LoadLibrary(libraryName);
+        }
+        else
+        {
+            var h = dlopen(libraryName, RTLD_LAZY);
```

Comment:
I bet our formatter will complain about the `var`. Also, it is preferrable to use full names like `handle` or `libHandle`.